sdk/eventhub/Azure.Messaging.EventHubs.Shared/src/Processor/PartitionLoadBalancer.cs

```diff
@@ -268,6 +269,14 @@ protected PartitionLoadBalancer()
             return claimedOwnership;
         }
 
+        /// <summary>
+        /// Returns <code>DateTimeOffset.UtcNow</code>, is used to make testing deterministic.
+        /// </summary>
+        internal virtual DateTimeOffset GetDateTimeOffsetNow()
```

Comment:
pedantic formatting nit:  Please move after the `public` members and before the `private` ones.